cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -174,8 +174,8 @@ private boolean isKafkaAssignerMode(Collection<String> goals) {
    * @param requirements The cluster model completeness requirements.
    * @param operationProgress The progress to report.
    * @param allowCapacityEstimation Allow capacity estimation in cluster model if the requested broker capacity is unavailable.
-   * @param concurrentPartitionMovements The maximum number of concurrent partition movements per broker
-   *                                     (if null, use num.concurrent.partition.movements.per.broker).
+   * @param concurrentInterBrokerPartitionMovements The maximum number of concurrent inter-broker partition movements per broker
```

Comment:
As mentioned in the previous comments, let's avoid changing the name of an existing config that is already in use as this will be a breaking change for users. Moreover, can we keep such name changes in their own patch rather than in a `4K+` patch?